src/app/directives/serial-modal.component/serial-modal.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
```

Comment:
Same question as with country select, are we using this? if so, great leave as is.